### ISSUE TITLE
Feature: Print the test cases list that are going to be executed

### ISSUE THREAD

**Issue by estemendoza** [argos83/marvin#31]
It would be helpful to have a command line option like "--dry-test" or "--no-run", so this way Marvin will print the list of test cases that would have been executed. It's helpful to try which test cases matches certain tags or how many test cases have certain tags, etc.

**Comment by argos83** [comment]
I agree, that could be helpful. What level of granularity do you think it should have? Should it print every iteration too? (I think the command switch should be `--dry-run`)

**Comment by estemendoza** [comment]
Yeah, `--dry-run` is the one I was looking for.

Yeah, it should print the iterations too since they can have tags and names and maybe only certain iterations are executed according to the selected tags
